.gitlab-ci.yml

```diff
@@ -2382,8 +2382,11 @@ build_agent7_windows1809:
     - cp ${OMNIBUS_PACKAGE_DIR}/datadog-agent-7*-x86_64.zip ${BUILD_CONTEXT}/datadog-agent-7-latest.amd64.zip
     - powershell -Command "docker build ${BUILD_ARG} --pull --file ${BUILD_CONTEXT}/windows/amd64/Dockerfile --tag ${TARGET_TAG} ${BUILD_CONTEXT}"
     - docker push ${TARGET_TAG}
+    - docker rmi ${TARGET_TAG}
+  after_script:
+    - docker system prune
```

Comment:
I find it less readable to use these kinds of mixins, and more prone to bugs (since depending on the order ones might be overriding others), so I try to avoid them unless the copy-paste they save is big.